presto-product-tests/README.md

```diff
@@ -143,6 +143,12 @@ presto-product-tests/bin/run_on_docker.sh <profile> -t com.facebook.presto.tests
 presto-product-tests/bin/run_on_docker.sh <profile> -t sql_tests.testcases.system.selectInformationSchemaTables
 ```
 
+Profile specific tests are tests which runs only for that particular profile. It confirms if the profile is same as what is claims.
+For e.g. singlenode-kerberos-hdfs-impersonation - Test to check if kerberos is working with impersonation
+To run the kerberos/impersonation specific test cases
+presto-product-tests/bin/run_on_docker.sh <profile> -g <profile>
```

Comment:
Other commands in this file are enclosed in triple ticks so please do that whenever you write a command. You can also inline commands when you don't want to give them special attention by using single ticks.